composer.json

```diff
@@ -20,46 +20,39 @@
         }
     ],
     "require": {
-        "php": ">=7.2",
-        "composer/composer": "^1.10.22",
-        "composer/installers": "^1.7",
+        "php": ">=7.3",
```

Comment:
On https://github.com/UN-OCHA/pocam8-site/pull/78, cafuego suggested moving to php7.4 at the same time.